jobs/io/ndb_io.py

```diff
@@ -74,32 +72,25 @@ def expand(self, pbegin):
 class PutModels(beam.PTransform):
     """Writes NDB models to the datastore."""
 
-    def __init__(self, datastoreio_stub, label=None):
-        """Initializes the PutModels PTransform.
-
-        Args:
-            datastoreio_stub: stub_io.DatastoreioStub. The stub instance
-                responsible for handling datastoreio operations.
-            label: str|None. The label of the PTransform.
-        """
-        super(PutModels, self).__init__(label=label)
-        self.datastoreio = datastoreio_stub
-
-    def expand(self, model_pcoll):
+    def expand(self, entities):
         """Writes the given models to the datastore.
 
+        This overrides expand from parent class.
```

Comment:
expand --> the expand() method

parent class --> the parent class

Please make similar changes elsewhere if needed.

Reply:
Done.